### PR TITLE
Submitting Proposal 3

### PR DESCRIPTION
**Pull request by Eoj-78**

### CODE REVIEW

File: Proposal-Template.md [hunk 1]
```diff
@@ -2,20 +2,51 @@
...
+
+Proposal 3 will address these concerns, along with the concerns that created the initial proposal that passed.
 
 ## Specification or Details
+Proposal 3 may consist of 6 separate votes, Part A - Part F.
+Part C - Part D will only be voted on if Part B passes.
+Part E - Part F will only be voted on if Part D passes.
+
+**Part A: Should we return DAO Allocation to Community Rewards?** YES or NO
+A YES vote on Part A will return the 25% of weekly rewards currently allocated to the DAO back to Community Rewards. A NO vote on Part A will leave the 25% of weekly rewards going toward the DAO.
```
Comment (thetavol) [inline Proposal-Template.md]: To be clear all of these funds are the DAOs. Perhaps we can rephrase for more clarity? Would something like the following work:

A YES vote on Part A means that none of the weekly community tokens will be set aside for later use, all will be allocated for use for rewards. A NO vote on Part A will leave 25% of the weekly community tokens being set aside for future use.

File: Proposal-Template.md [hunk 2]
```diff
@@ -2,20 +2,51 @@
...
+
+**Part B: Should we return to regular reward voting to set the reward allocations for SLP and Staked VOL?** YES or NO
```
Comment (thetavol) [inline Proposal-Template.md]: Let's flesh this out a bit. Can we add:

A YES vote means that the reward amounts for SLP and Staked VOL will be set by quadratic community votes. A NO vote means that SLP and Staked VOL reward %s will remain what they are now.

File: Proposal-Template.md (hunk 1 above)
Comment (Eoj-78) [reply Proposal-Template.md]: I made this change, it is under commits? I guess it has to be merged

File: Proposal-Template.md (hunk 2 above)
Comment (Eoj-78) [reply Proposal-Template.md]: Same.

**Pull request merged**